packages/components/src/select-control/index.native.js

```diff
@@ -17,7 +17,7 @@ function SelectControl( {
 	const id = `inspector-select-control-${ instanceId }`;
 
 	return (
-		<PickerCell
+		<CyclePickerCell
```

Comment:
Though I did not include it in this PR yet. I am proposing that we remove [PickerCell.](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/mobile/bottom-sheet/picker-cell.native.js)(though Picker is still super useful!). Though PickerCell gets the BottomSheet nested menu about 80% there, it feels like a fundamental restriction of a [react-native-modal](https://github.com/react-native-community/react-native-modal) Modal nested inside of another, that we will not be able to close the containing Modal before showing the child Modal without editing the underlying library. @Tug @etoledom thoughts? 

One reason this might be worth doing now is in order to decouple PickerCell from Cross Platform component refactoring efforts, like seen here in SelectControl.